guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -49,19 +49,37 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
 
         /**
          * The DOM element which will contain the clipboard contents within the
-         * user interface provided by this directive.
+         * user interface provided by this directive. We populate the clipboard
+         * editor via this DOM element rather than updating a model so that we
+         * have support for rich text contents.
```

Comment:
I think we're getting a bit ahead of ourselves claiming that we "have support for rich text contents". Might want to clarify that this is to enable future support for this functionality, since we certainly don't have it now.